src/main/java/org/mamute/controllers/RankingController.java

```diff
@@ -34,8 +34,8 @@ public void rank(Integer p) {
 	}
 	
 	@Get
-	public void tagRank(String tagName){
-		Tag tag = tags.findByName(tagName);
+	public void tagRank(String tagUriName){
```

Comment:
Better not to change controllers methods parameter's names. They are used at [`routes.properties`](https://github.com/caelum/mamute/blob/master/src/main/resources/routes.properties) files and someone could overwrite those.